### PR TITLE
Update homepage content

### LINKED ISSUE #60: /homepage build new updated homepage

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated the homepage content.

## QA

 ./run and check the homepage against the [copydoc](https://docs.google.com/document/d/1vIsFwNOjFu3E-_RPDi6ArArDSMHbYaWSTmy0BmKLhS0/edit#)

## Issue / Card

Fixes #60

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -65,17 +65,15 @@ body-class: "home"
     </div>
     <div class="col-3">
       <h3><a href="web-style-guide">Web&nbsp;&rsaquo;</a></h3>
-      <p>Find out about the Vanilla Framework, created to ease design and development of Ubuntu and Canonical sites, and free to use on your own projects too.</p>
+      <p>Find out about the Vanilla Framework, created to ease design and development of Ubuntu and Canonical sites, and free to use on your own projects&nbsp;too.</p>
```
Comment (barrymcgee) [inline index.md]: "Find out about the Vanilla Framework" - we never really prepend VF with 'the'

File: index.md [hunk 2]
```diff
@@ -65,17 +65,15 @@ body-class: "home"
...
     </div>
     <div class="col-3">
       <h3><a href="font">Font&nbsp;&rsaquo;</a></h3>
-      <p>Learn more about the Ubuntu font family, the available fonts, weights and languages, and how to get it for free.</p>
+      <p>Learn more about the Ubuntu font family, the available fonts, weights and languages, and how to get it for&nbsp;free.</p>
```
Comment (barrymcgee) [inline index.md]: Why are you using non-breaking space entities? To stop the last word from being orphaned on a new line? The flip side of this is that you then get overly generous whitespace on the penultimate line where the penultimate word should be..

Comment (deadlight) [reply index.md]: Yep, the orphans looked much worse than the extra whitespace so I made a call. The whole build will be design reviewed where these assumptions can be confirmed.

File: index.md (hunk 1 above)
Comment (deadlight) [reply index.md]: Sure, I'll update.

**Comment by deadlight** [comment]
@barrymcgee I've removed the "the vanilla framework" reference

**Pull request merged**